core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthCheckUpdateHandler.java

```diff
@@ -28,6 +28,14 @@
  */
 @FunctionalInterface
 public interface HealthCheckUpdateHandler {
+
+    /**
+     * Returns the default {@link HealthCheckUpdateHandler} implementation.
+     */
+    static HealthCheckUpdateHandler ofDefault() {
```

Comment:
We sometimes use `ofDefault()` and sometimes `of()`. Maybe it's time to clean up a little bit? Which one would be better?

Reply:
I think that it is best to distinguish `of` and `ofDefault` by whether creating new instance. If `of`, create a new instance with default settings, and `ofDefault`, simply returns a singleton instance.